addons/bus/models/res_partner.py

```diff
@@ -9,12 +9,14 @@ class ResPartner(models.Model):
     _inherit = 'res.partner'
 
     im_status = fields.Char('IM Status', compute='_compute_im_status')
+    last_seen = fields.Datetime('User last Seen', compute='_compute_im_status')
```

Comment:
ok, after thinking about it, and since it is not on the specs, I think that this field should not be added.  The last seen information is not really important in the discuss interface.  If we want to add it, this should go in another task.  Please remove the last_seen field from this PR